app/components/listing/feedback/FeedbackModal.scss

```diff
@@ -0,0 +1,59 @@
+.feedback-modal-body {
+  letter-spacing: 0.402722px;
+  .close-modal {
+    position: absolute;
+    right: 5%;
+    cursor: pointer;
+  }
+  .feedback-header {
+    font-weight: 600;
+    font-size: 20px;
+    line-height: 27px;
+    text-align: center;
+    color: #1c72e6;
+  }
+  .feedback-subheader {
+    margin: 0 auto;
+    width: 203.42px;
+    font-size: 14.498px;
+    line-height: 20px;
+    text-align: center;
+  }
+  .vote-header {
+    margin-top: 30px;
+    font-weight: 600;
+    font-size: 18px;
+    line-height: 25px;
+    text-align: center;
+  }
+  .vote-icons {
+    width: 40%;
+    height: 35px;
+    margin: 30px auto;
+    display: flex;
+    justify-content: space-evenly;
```

Comment:
I think it's probably harder to reason about the actual spacing between the icons if you use `space-evenly`, since you have to think about the spare around the outsides. I think `space-between` is probably easiest to use, since it pushes the icons to the outsides.

Just as a quick demonstration as to why `space-evenly` is harder to think about: You would first take the width of the modal (577px), then multiple by the width of the `vote-icons` (40%), then subtract the width of the icons themselves (34px * 2), and then divide the remaining space by 3 for the three areas of spacing to get the distance between the two icons. Now imagine that you have to solve this in reverse in order to get the icons to be spaced a specific distance apart.

With `space-between`, it's slightly simpler because instead of having to divide by 3, the totality of the remainder space is placed between the icons.

As an alternative, you may even consider just putting margins between the icons, which make it even easier to control the spacing between them.